man/shadow.5.xml

```diff
@@ -177,19 +173,16 @@
 	</term>
 	<listitem>
 	  <para>
-	    The number of days after a password has expired (see the
-	    maximum password age above) during which the password should
-	    still be accepted (and the user should update her password
-	    during the next login).
+	    The number of days after a password expires (see the maximum password
+	    age above) during which the password is still accepted,
```

Comment:
Please put parentheticals in their own lines (semantic newlines).

Reply:
ok